.circleci/config.yml

```
@@ -67,4 +67,7 @@ workflows:
           context: SonarCloud
       - deploy:
           requires:
-            - test
\ No newline at end of file
+            - test
```

yamllint

[warning] wrong indentation: expected 10 but found 12

(indentation)